src/index.js

```diff
@@ -47,7 +47,7 @@ class Dotenv {
     }
 
     Object.keys(blueprint).map(key => {
-      const value = env[key]
+      const value = env[key] || vars[key]
```

Comment:
In that case the only falsey value to consider is "", empty string.

If a system variable is set, can the dotenv override it with an empty value?